nvflare/fuel/f3/streaming/byte_streamer.py

```diff
@@ -51,10 +51,21 @@
 
 class TxTask:
     def __init__(
-        self, channel: str, topic: str, target: str, headers: dict, stream: Stream, secure: bool, optional: bool
+        self,
```

Comment:
Not sure about this seemingly massive changes. Why are these necessary?

Reply:
Removed the changes from 2.5.